### PR TITLE
Template tags and filters for tenants

### PR DESCRIPTION
**Pull request by jfavellar90**
Adding template tags and filter with more meaningful names, and adding deprecation notes for methods including the name "microsite"

@morenol 
@anmrdz 
@cocococosti

### CODE REVIEW

File: eox_tenant/templatetags/ednx.py [hunk 1]
```diff
@@ -154,3 +158,49 @@ def get_login_link():
...
+@register.simple_tag(name="tenant_rtl")
+def tenant_rtl_tag():
+    """
+    Django template tag that outputs the direction string for rtl support
+    """
+    return 'rtl' if get_language_bidi() else 'ltr'
+
+
+@register.filter
+def tenant_template_path(template_name):
```
Comment (morenol) [inline eox_tenant/templatetags/ednx.py]: I am not sure if we need this function

Comment (jfavellar90) [reply eox_tenant/templatetags/ednx.py]: @morenol now I'm using the get_template_path method from theming helpers which is a proxy to hide microsite.get_template_path behind comprehensive theming

File: eox_tenant/templatetags/ednx.py [hunk 2]
```diff
@@ -154,3 +160,48 @@ def get_login_link():
...
+    """
+    Django template filter to apply template overriding to microsites
+    """
+    return theming_helpers.get_template_path(template_name)
+
+
+@register.simple_tag
+def tenant_get_value(value, *args, **kwargs):  # pylint: disable=unused-argument
+    """
+    Django template filter that wrapps the configuration_helpers.get_value function
```
Comment (anmrdz) [inline eox_tenant/templatetags/ednx.py]:
```suggestion
    Django template filter that wraps the configuration_helpers.get_value function
```

File: eox_tenant/templatetags/ednx.py [hunk 3]
```diff
@@ -60,14 +64,16 @@ def microsite_rtl_tag():
...
     """
-    return microsite.get_template_path(template_name)
+    return theming_helpers.get_template_path(template_name)
 
 
 @register.simple_tag
 def microsite_get_value(value, *args, **kwargs):  # pylint: disable=unused-argument
     """
-    Django template filter that wrapps the microsite.get_value function
+    Django template filter that wrapps the configuration_helpers.get_value function
```
Comment (anmrdz) [inline eox_tenant/templatetags/ednx.py]:
```suggestion
    Django template filter that wraps the configuration_helpers.get_value function
```

File: eox_tenant/templatetags/ednx.py (hunk 2 above)
Comment (jfavellar90) [reply eox_tenant/templatetags/ednx.py]: Done

File: eox_tenant/templatetags/ednx.py (hunk 3 above)
Comment (jfavellar90) [reply eox_tenant/templatetags/ednx.py]: Done

**Pull request merged**